Makefile

```diff
@@ -34,16 +36,20 @@ all: build
 build:
 	$(GO_BUILD_RECIPE)
 
+test:
+	go test -race -v -cover=1 ./cmd/... ./pkg/... -coverprofile $(COVERPROFILE)
+
 test-e2e: 
-	@${TEMPLATE_CMD} manifests/0110_namespace.yaml > manifests/operator-init.yaml
-	echo -e "\n---\n" >> manifests/operator-init.yaml
+	@${TEMPLATE_CMD} manifests/0100_namespace.yaml > manifests/operator-init.yaml
```

Comment:
Only fix the 0100_namespace.yaml in this PR

Reply:
working on it

Reply:
> 64b41f0